notebooks/02-Introduction.ipynb

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
<p>Mention how this is currently supported using the <code>astropy.modeling</code> package </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>done I think.</p>